client/ui.js

```diff
@@ -147,7 +147,7 @@
       while (target && !(id = target.id)) {
         target = target.parentNode;
       }
-      index = id && --id.split('-')[1] || 0;
+      index = (id && --id.split('-')[1]) || 0;
```

Comment:
I made this change to appease eslint. the rest of the changes in this file are from `eslint --fix`